### PR TITLE
Fix scrolling feat23

### PR DESCRIPTION
**Pull request by leftynaut**
#23 

- will create new bug issue for advanced mobile scrolling implementation

### CODE REVIEW

File: client/public/js/whiteboard.js [hunk 1]
```diff
@@ -19,48 +30,58 @@ const newCanvas = function(){
...
-	let start = function(e) {
-		ctx.beginPath();
-		x = e.changedTouches[0].pageX;
-		y = e.changedTouches[0].pageY - 34;
-		ctx.moveTo(x, y);
+
+	const startTouch = function(e) {
+    // if (e.targetTouches.length < 2) {
+      // e.preventDefault();
+  		ctx.beginPath();
```
Comment (shamsup) [inline client/public/js/whiteboard.js]: Tabs to spaces

File: client/public/js/whiteboard.js [hunk 2]
```diff
@@ -19,48 +30,58 @@ const newCanvas = function(){
...
+  		x = e.changedTouches[0].pageX + content.scrollLeft;
```
Comment (shamsup) [inline client/public/js/whiteboard.js]: `const x = ...` 

The variables `x` and `y` are never defined
`content` here is undefined unless I'm missing a global variable

File: client/public/js/whiteboard.js [hunk 3]
```diff
@@ -19,48 +30,58 @@ const newCanvas = function(){
...
+  		y = e.changedTouches[0].pageY - 34 + content.scrollTop;
```
Comment (shamsup) [inline client/public/js/whiteboard.js]: `const y = `

File: client/public/js/whiteboard.js [hunk 4]
```diff
@@ -19,48 +30,58 @@ const newCanvas = function(){
...
-		x = e.changedTouches[0].pageX;
-		y = e.changedTouches[0].pageY - 34;
-		ctx.lineTo(x, y);
-		ctx.stroke();
+	const moveTouch = function(e) {
+    // if (e.targetTouches.length < 2) {
+    if (canvasState) {
+      e.preventDefault();
+      x = e.changedTouches[0].pageX + content.scrollLeft;
+      y = e.changedTouches[0].pageY - 34 + content.scrollTop;
```
Comment (shamsup) [inline client/public/js/whiteboard.js]: `const x =`
`const y =`

File: npm-debug.log [hunk 5]
```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```
Comment (shamsup) [inline npm-debug.log]: Remove this file from your commit

File: client/public/js/whiteboard.js [hunk 6]
```diff
@@ -19,48 +30,58 @@ const newCanvas = function(){
...
+      ctx.lineTo(x, y);
+      ctx.stroke();
+    }
+    // }
 	};
-  document.getElementById('canvas').addEventListener('touchstart', start, false);
-	document.getElementById('canvas').addEventListener('touchmove', move, false);
+
+  canvas.addEventListener('touchstart', startTouch, false);
+	canvas.addEventListener('touchmove', moveTouch, false);
```
Comment (shamsup) [inline client/public/js/whiteboard.js]: I believe `canvas` here is undefined unless I'm missing a global variable somewhere

File: client/public/js/whiteboard.js (hunk 1 above)
Comment (leftynaut) [inline client/public/js/whiteboard.js]: fixed (i believe)

File: client/public/js/whiteboard.js (hunk 2 above)
Comment (leftynaut) [inline client/public/js/whiteboard.js]: fixed for xy. content is defined in outer createCanvas function

File: client/public/js/whiteboard.js (hunk 3 above)
Comment (leftynaut) [inline client/public/js/whiteboard.js]: fixed

File: client/public/js/whiteboard.js (hunk 4 above)
Comment (leftynaut) [inline client/public/js/whiteboard.js]: fixed

File: client/public/js/whiteboard.js (hunk 6 above)
Comment (leftynaut) [inline client/public/js/whiteboard.js]: canvas is defined in outer createCanvas function

File: npm-debug.log (hunk 5 above)
Comment (leftynaut) [inline npm-debug.log]: removed

File: client/public/js/whiteboard.js [hunk 7]
```diff
@@ -1,11 +1,7 @@
 let ctx, color = '#000';
 let canvasState = true;
 const moveCanvas = function() {
-  if (canvasState) {
-    canvasState = false;
-  } else {
-    canvasState = true;
-  }
+  canvasState ^= true;
```
Comment (shamsup) [inline client/public/js/whiteboard.js]: I'm not sure if you realize this, but it actually converts `canvasState` from a boolean to an integer (`1` or `0`)

File: client/public/js/whiteboard.js [hunk 8]
```diff
@@ -1,66 +1,79 @@
 let ctx, color = '#000';
+let canvasState = true;
```
Comment (shamsup) [inline client/public/js/whiteboard.js]: define `canvas` and `content` in this scope as well

**Pull request merged**